scripts/cleanup.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+cd "${DIR}/.."
+rm -r node_modules/
+rm yarn.lock
```

Comment:
`yarn.lock` should never be deleted as part of a cleanup. It should be used in exactly the same form as it is checked in.